kolibri/core/assets/test/exams/utils.spec.js

```diff
@@ -150,20 +150,19 @@ describe('exam utils', () => {
       expect(converted[0].learners_see_fixed_order).toEqual(false);
     });
     it('always sets the question_count and learners_see_fixed_order properties to the original exam values', () => {
-      exam.learners_see_fixed_order = false;
-      exam.question_count = 49;
-      const converted = convertExamQuestionSourcesToV3(exam);
-      expect(converted).toEqual([
-        {
-          section_id: converted[0].section_id,
-          section_title: '',
-          description: '',
-          resource_pool: [],
-          questions: expectedSources.sort(),
-          learners_see_fixed_order: false,
-          question_count: 49,
-        },
-      ]);
+      // exam.learners_see_fixed_order = false;
+      // exam.question_count = 49;
+      // const converted = convertExamQuestionSourcesToV3(exam);
+      // expect(converted).toEqual([
+      //   {
+      //     section_title: '',
+      //     description: '',
+      //     resource_pool: [],
+      //     questions: expectedSources.sort(),
+      //     learners_see_fixed_order: false,
+      //     question_count: 49,
+      //   },
+      // ]);
```

Comment:
This might just be due to some work in progress adjusting as you're thinking things through, but let's just not remember to accidentally commit this commented out test when the PR is merged. I'm not sure if the test needs to be removed or updated as the work gets more finalized.